tests/phpunit/tests/test-wpml-gutenberg-string-registration.php

```diff
@@ -426,3 +502,18 @@ private function getStringsInBlock( $config_option ) {
 		return new WPML\PB\Gutenberg\StringsInBlock\Collection( $string_parsers );
 	}
 }
+
+class ClosureMock {
```

Comment:
**Note:** I couldn't find a standard way of mocking a closure so I created this class to help. I plan to move this to a separate test library when I introduce the Functional library into WPML.